_data/other.yml

```diff
@@ -63,6 +63,13 @@ websites:
     twitter: bananatag
     facebook: trackemail
 
+  - name: Baremetrics
+    url: https://baremetrics.com/
+    img: baremetrics.svg
+    tfa:
+      - totp
+    doc: https://changelog.baremetrics.com/two-factor-authentication-3X2v9m
```

Comment:
This page doesn't provide any meaningful information, The screenshot is much more helpful.
```suggestion
```